module/configs/pych.json

```diff
@@ -102,7 +102,7 @@
             ],
 
             "commands": [
-                "chpl %(cflags)s %(ctargets)s -o __TMP_PATH__",
+                "chpl %(cflags)s __DEPENDS__ %(ctargets)s -o __TMP_PATH__",
```

Comment:
I'm of the mind that if the user is throwing Chapel flags in left and right, they probably should just be writing Chapel code.  Most of our flags feel too tweaky to me rather than essential for functionality and pyChapel feels like minor experimentation or looking for a quick performance boost rather than committing to truly understanding the language or its many flags.  But I'm willing to be convinced otherwise :)